library/pod-security-policy/allow-privilege-escalation/template.yaml

```diff
@@ -3,7 +3,9 @@ kind: ConstraintTemplate
 metadata:
   name: k8spspallowprivilegeescalationcontainer
   annotations:
-    description: Controls restricting escalation to root privileges.
+    description: >-
+      Controls restricting escalation to root privileges. Corresponds to the
+      `allowPrivilegeEscalation` field in a PodSecurityPolicy.
```

Comment:
For PSP specifically, the main purpose is to provide guidance for people who are referencing the PSP fields/ documentation.

The PSP documentation references its fields by non-fully-qualified name:

https://kubernetes.io/docs/concepts/policy/pod-security-policy/

So we should follow their conventions.

Since PSP is deprecated, we may choose to remove these references entirely once most/all users are no longer in a transitional phase.

If we are talking about related fields within the same constraint template, that seems reasonable to me, though I'd be a bit concerned about verbosity. Maybe make all paths relative to `parameters`?

Reply:
I'm fine either way.

Reply:
I lean more towards matching the PSP docs (meaning what's written is fine as-is)